src/openrct2/ride/Vehicle.h

```diff
@@ -251,7 +251,7 @@ struct Vehicle : SpriteBase
         int16_t SwingSpeed;
         int16_t crash_z;
     };
-    VEHICLE_STATUS status;
+    Vehicle::Status status;
```

Comment:
```suggestion
    Status status;
```
No need to specify the scope here.